schema_base_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+/* TABLES */
+
+CREATE TABLE patients (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100) NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+  patient_id INT NOT NULL,
+  status VARCHAR(120) NOT NULL,
+  CONSTRAINT fk_patient_id FOREIGN KEY (patient_id) REFERENCES patients(id),
+);
```

Comment:
- [ ] Kindly remove the **_comma(,)_** at the end of this PSQL statement which creates a constraint in the `medical_histories` table. This causes a syntax error when I try to run this file. 👍 💻 

![image](https://github.com/hernandanielzamora/Clinic/assets/77106962/6202f26c-4532-4cc5-8291-f35ee725d41a)